tools/gen_openvino_dockerfile.py

```diff
@@ -121,22 +123,22 @@ def dockerfile_for_linux(output_file):
         ln -s libopenvino_c.so.${OPENVINO_VERSION} libopenvino_c.so && \
         ln -s libopenvino_ir_frontend.so.${OPENVINO_VERSION} libopenvino_ir_frontend.so.${OV_SHORT_VERSION} && \
         ln -s libopenvino_ir_frontend.so.${OPENVINO_VERSION} libopenvino_ir_frontend.so)
-'''
+"""
 
-    df += '''
+    df += """
```

Comment:
we have been using ''' for all the dockerfiles. 
eg: https://github.com/triton-inference-server/server/blob/main/build.py#L891

can we keep this consistent?

Reply:
Thanks so much for your review! The server repo PR actually enforces the same changes, including for build.py: https://github.com/triton-inference-server/server/pull/6022/files

This has actually been the most controversial feature in Black, since different engineers have different preferences on single vs double quotes. Their logic is [here](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings). Essentially, Black prefers double quotes over single quotes unless introducing them results in more escape characters. The main benefit with Black being opinionated and not configurable is that people don't have to debate about these kind of formatting rules.

However, this was their most-contested issue (you can see the deep [debate/discussion](https://github.com/psf/black/issues/118) here). They acquiesced and created an option to skip normalization via `--skip-string-normalization`. It's not recommended and most teams do not use it. I feel like we should do the same and it would satisfy the requirement of being consistent. If you think it's worth discussion though, happy to start a team discussion and see what would make sense for everyone.